### DISCUSSION TITLE
View

### DISCUSSION THREAD

**Discussion by Quantum-Codes** [Sid72020123/scratchconnect#20] [Q&A] [answered]
https://github.com/Sid72020123/scratchconnect/blob/bf4a51101a624b005ef401299e416d141296005c/scratchconnect/Project.py#L382

What does this do? Return no. Of views or what?

**Comment by Sid72020123** [comment, marked as answer]
It just views the project or increases the view count by one. There are other functions to get project stats!

**Reply by Quantum-Codes** [reply]
VIEWBOTTING?!?!?!??!?!??!?!!??!?!!?!

**Reply by Sid72020123** [reply]
No it's a API to just view or increase the view count by 1. I think a user can't use this more than once.

**Reply by Quantum-Codes** [reply]
VIEWBOTS EXIST!!!! 1 PROJECT WAS GONNA SURPASS GEOMETRY DASH!!!!! wasn't even good project. THEY VIEWBOTTED

**Reply by Sid72020123** [reply]
Oh! LOL

**Reply by Sid72020123** [reply]
But I think Scratch API only allow to increase view count by 1 but if more than 1 then this bug should be reported to ST!

**Reply by Quantum-Codes** [reply]
Lol, ok. HenryWolfe on YouTube posted a video on that (Scratch user= HenryWolf or Wolfe idk lol)

**Reply by Sid72020123** [reply]
LOL Ok.
